modules/sandboxed-containers-installing-operator.adoc

```diff
@@ -0,0 +1,7 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="installing-sandboxed-container-operator_{context}"]
+
+= Installing the Sandboxed Containers Operator
```

Comment:
You can use your `Sandboxed Containers Operator` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title